hardware/Makefile

```diff
@@ -48,6 +48,12 @@ python          ?= python3
 # Enable tracing
 snitch_trace    ?= 1
 
+# Path to DRAMsys
+dramsys_resouces_path ?= $(MEMPOOL_DIR)/hardware/deps/dram_rtl_sim/dramsys_lib/DRAMSys/configs
+dramsys_lib_path ?= $(MEMPOOL_DIR)/hardware/deps/dram_rtl_sim/dramsys_lib/DRAMSys/build/lib
+questa_args += +DRAMSYS_RES=$(dramsys_resouces_path)
+questa_args += -sv_lib $(dramsys_lib_path)/libsystemc -sv_lib $(dramsys_lib_path)/libDRAMSys_Simulator
```

Comment:
Would supporting VCS and DRAM-sys be much effort for a future PR?

Reply:
I tried, but it did not work well because the GCC version of `.so` was mismatched. I think we can leave it for the future PR.

Reply:
Yes, sure.